src/tests.c

```diff
@@ -1,4 +1,7 @@
 #include "hashmap.h"
+#include <assert.h>
+#include <stdio.h>
+#include <string.h>
```

Comment:
Best to put includes in the file theyre used in. That way you don't have to rely on other files not changing for the current one to keep working